ONSdigital/eq-survey-runner
# eq-324 additional paradata

### Changes
- added `analytics.js` module to interface with google analytics event tracking
- added hooks in template with data to be sent to analytics
- fires events according to errors generated
- fires events when user clicks guidance toggle
### How to test
- `npm run compile`
- visit a page and submit with errors
- verify events are being caught by analytics
- click on guidance toggle
- verify events are being caught by analytics
### Who can test

@bennpearson
